### DISCUSSION TITLE
Rclone use the service account

### DISCUSSION THREAD

**Discussion by Namlee123** [anasty17/mirror-leech-telegram-bot#1409] [General]
Hello Anasty17!
Let's say I use rclone to upload to GDrive.
By default rclone.org does not automatically change SA (100 SA) when the limit of 750GB/day/sa is reached.

I want to ask you about the rclone feature in the bot that supports automatically changing SA to exceed the limit?
If that feature is available, how do I configure rclone.

Here is my current rclone configuration using SA:

[da1]
type = drive
scope = drive
service_account_file = /accounts/00.json
team_drive = XXX..XXX

**Comment by anasty17** [comment]
No no auto switch i still thinking of adding it we already have perfect switching using python google api rclone mainly for other clouds

**Reply by anasty17** [reply]
> Hello Anasty17! Let's say I use rclone to upload to GDrive. By default rclone.org does not automatically change SA (100 SA) when the limit of 750GB/day/sa is reached.
> 
> I want to ask you about the rclone feature in the bot that supports automatically changing SA to exceed the limit? If that feature is available, how do I configure rclone.
> 
> Here is my current rclone configuration using SA:
> 
> [da1] type = drive scope = drive service_account_file = /accounts/00.json team_drive = XXX..XXX

This added
